.storybook/main.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('@storybook/react/types').StorybookConfig} */
+module.exports = {
+  stories: ["../easy-ui-react/src/**/*.stories.mdx"],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+  ],
```

Comment:
question: do actions need to be included here as well?

suggestion: what are your thoughts on the viewport addon https://storybook.js.org/docs/react/essentials/viewport ?